Makefile

```diff
@@ -11,52 +12,59 @@ favicon:
 website:
 	LONG_CACHE=false npm run build
 
-# `website` is needed for the server deploys.
-deploy: website deploy-s0 deploy-s1 deploy-s2 deploy-gh-pages deploy-gh-pages-clean
+deploy: deploy-s0 deploy-s1 deploy-s2 clean-server-deploy deploy-gh-pages deploy-gh-pages-clean
 
-deploy-s0:
+deploy-s0: prepare-server-deploy push-s0
+deploy-s1: prepare-server-deploy push-s1
+deploy-s2: prepare-server-deploy push-s2
```

Comment:
All those are meant to be `.PHONY`, right?

Reply:
Ah yes, thanks.